### PR TITLE
Update the .NET readme.md, update Xamarin iOS sample.

### PR DESCRIPTION
**Pull request by bradserbuddy**

### CODE REVIEW

File: Samples/iOS/BuddySquare/AppDelegate.cs [hunk 1]
```diff
@@ -27,24 +28,27 @@ public partial class AppDelegate : UIApplicationDelegate
...
             Current = this;
+
+			this.locationManager = new LocationManager ();
         }
 
 		// TODO: Go to http://dev.buddyplatform.com to get an app ID and app key.
-		private const String APP_ID = "\Your App ID";
-		private const String APP_KEY = "\Your App Key"; 
+		private const String APP_ID = "bbbbbc.hKpFwzglHBrK";
+		private const String APP_KEY = "616bf446-f005-fe9a-9fd4-e62ba8913bf5";
```
Comment (xyziemba) [inline Samples/iOS/BuddySquare/AppDelegate.cs]: Are the id and key intentionally left in because this is a sample?

Comment (bradserbuddy) [inline Samples/iOS/BuddySquare/AppDelegate.cs]: Fixed.

**Comment by davesters** [comment]
nit: The tabbing looks all wonky. Seems like has mixed tabs and spaces.

**Comment by bradserbuddy** [comment]
I'll take a look at that... It looks fine in the Xamarin IDE.

**Comment by davesters** [comment]
Looks fine to me

**Pull request merged**